wsf_scraping/spiders/msf_spider.py

```diff
@@ -0,0 +1,69 @@
+import scrapy
+from .base_spider import BaseSpider
+from wsf_scraping.items import MSFArticle
+
+
+class MSFSpider(BaseSpider):
+    name = 'msf'
+
+    custom_settings = {
+        'JOBDIR': 'crawls/msf'
+    }
+
+    def start_requests(self):
+        """Set up the initial request to the website to scrape."""
+
+        urls = [
+            'https://www.msf.org.uk/activity-reports',
+            'https://www.msf.org.uk/reports',
+        ]
+
+        for url in urls:
+            self.logger.info('Initial url: %s', url)
+            yield scrapy.Request(
+                url=url,
+                errback=self.on_error,
+                dont_filter=True,
+                callback=self.parse,
+            )
+
+    def parse(self, response):
+        """ Parse both reports and activity-reports pages.
+
+        @url https://www.msf.org.uk/activity-reports
+        @returns items 0 0
+        @returns requests 10
+        """
+
+        doc_links = response.css('.field-items a::attr(href)').extract()
```

Comment:
it seems that you can create a base class that contains this method that needs `'.field-items a::attr(href)'` to run. is that the case? if so it would be good to refactor at this point.